### PR TITLE
translated api-files-gatsby-node.md

### PR DESCRIPTION
**Pull request by noisefilter19**

### CODE REVIEW

**Comment by monizb** [comment]
@arshadkazmi42 Should i take this for a first look?

**Comment by arshadkazmi42** [comment]
@monizb sure go ahea

File: docs/docs/api-files-gatsby-node.md [hunk 1]
```diff
@@ -1,18 +1,18 @@
...
 ---
 
-Code in the file `gatsby-node.js` is run once in the process of building your site. You can use it to create pages dynamically, add nodes in GraphQL, or respond to events during the build lifecycle. To use the [Gatsby Node APIs](/docs/node-apis/), create a file named `gatsby-node.js` in the root of your site. Export any of the APIs you wish to use in this file.
+फ़ाइल में कोड `gatsby-node.js` आपकी साइट के निर्माण की प्रक्रिया में एक बार चलाया जाता है। आप इसका उपयोग गतिशील रूप से पेजस बनाने के लिए, GraphQL में नोड्स जोड़ने के लिए या बिल्ड लाइफसाइकिल के दौरान घटनाओं पर प्रतिक्रिया देने के लिए कर सकते हैं। [Gatsby Node APIs] (/docs/node-apis/) का उपयोग करने के लिए, अपनी साइट के रूट में `gatsby-node.js` नामक फ़ाइल बनाएं। इस फ़ाइल में आपके द्वारा उपयोग किए जाने वाले किसी भी API को एक्सपोर्ट करें।
 
-Every Gatsby Node API passes a [set of Node API helpers](/docs/node-api-helpers/). These let you access several methods like reporting, or perform actions like creating new pages.
+हर Gatsby नोड API passes a [नोड एपीआई सहायकों का एक सेट](/docs/node-api-helpers/). ये आपको रिपोर्टिंग जैसे कई तरीकों तक पहुँचने देते हैं, या नए पेज बनाने जैसे कार्य करने देता है ।
```
Comment (monizb) [inline docs/docs/api-files-gatsby-node.md]:
```suggestion
हर Gatsby नोड API एक [नोड एपीआई सहायकों का एक सेट] पास करता है (/docs/node-api-helpers/)। ये आपको कई तरीकों तक पहुँचने देते हैं जैसे रिपोर्टिंग, नए पेज बनाने जैसे कार्य करने देता है ।
```

**Review by monizb — CHANGES_REQUESTED** [review]
@noisefilter19 Thanks for the translation, there is a small error after which we should be good to go!

**Comment by arshadkazmi42** [comment]
@noisefilter19 any updates here?

**Comment by noisefilter19** [comment]
Sorry, was caught up with some work.
Have made the changes

**Review by arshadkazmi42 — APPROVED** [review]
👍 

Thank you for working on this

**Comment by arshadkazmi42** [comment]
@monizb can you approve this PR. So i can merge it

**Pull request merged**